### ISSUE TITLE
What username is meant?

### ISSUE THREAD

**Issue by Strit** [JupiterBroadcasting/jupiterbroadcasting.com#280]
In the `contents/people/*.md` pages there's a field simply called `username`.

There is no indication of what platform the username is for. Is it Mumble, Matrix, IRC?

So maybe we can change the field name into a specific name, like `mumble_username`?

**Comment by Strit** [comment]
Looking at the code in [show-scraper](https://github.com/JupiterBroadcasting/show-scraper/blob/main/models/person.py#L9), it looks like it might just be an entity to link people to [shows](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/main/content/show/linux-unplugged/0470.md?plain=1#L79) and not actually a username of any kind.

Am I understanding it correctly?

**Comment by kbondarev** [comment]
It is just a unique ID of each person, that is also used the URL slug.

I guess `slug`, `person_slug`, or `GUID` would've been a better name for this parameter

**Comment by kbondarev** [comment]
Speaking of, I noticed that it is on the frontend as "@username" under the full name of the arch guest/host.
Probably shouldn't be displayed there like that, as it might make ppl thing it's an username on social media (like twitter or Instagram)

**Comment by Strit** [comment]
Yeah, that's why I got confused when I saw my "username", but I have no social media with that handle.

**Comment by elreydetoda** [comment]
I definitely think we could change they key name for v2 of the JB.com, but till then we can just remove the `@username` reference in the guest/host template (IIRC, one of them should just be a symlink to the other after my PR that merged for #110)

**Comment by ironicbadger** [comment]
What is the output of the scraper in this field?

**Comment by gerbrent** [comment]
https://github.com/JupiterBroadcasting/show-scraper/blob/main/models/person.py#L9

**Comment by gerbrent** [comment]
https://raw.githubusercontent.com/JupiterBroadcasting/jupiterbroadcasting.com/main/content/people/alex.md

**Comment by ironicbadger** [comment]
https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/main/content/people/allanjude.md?plain=1#L12

**Comment by ironicbadger** [comment]
So looking at this we can probably call it whatever we want. 

`person_uuid` might be adequate?

**Comment by elreydetoda** [comment]
Ya, the only snag is that if we change it in the scraper then we need to update it every place it's referenced in the main JB site as well. So, that's why I was suggesting the v2 release instead of changing something so vital this close to us trying to do v1 release.
